mantidproject/mantid
# 29143 Disabled fit tab on muon guis when no data present

**Description of work.**
Fit tab was disabled when there was no data present for fitting tab to use


**Report to:** Adrian

**To test:**
1. Open Muon analysis gui 
2. Drag fitting table out.
3. Load data and fitting tab should be active.

4. Close Muon analysis gui and open Frequency domain analysis gui.
5. Repeat step 2 and load data.
6. Go to the transform tab and click on calculate FTT or MaxEnt after calculation is done fitting tab should be active.




Fixes #29143 .



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
